build.gradle

```
@@ -8,7 +8,7 @@ buildscript {
     jcenter()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.2.0'
+    classpath 'com.android.tools.build:gradle:4.1.0'
```

It looks like 4.1.1 came out this month (probably after you'd started working on this). Seems reasonable to go for the latest while we're in here changing it already.
```suggestion
    classpath 'com.android.tools.build:gradle:4.1.1'
```